### PR TITLE
Testing flask framework with pytest

### PR DESCRIPTION
**Pull request by MarkusHackspacher**
#3 
a simple test.

```
tests/test_api.py ...                                                    [100%]
============================== 3 passed in 1.32s ===============================
```

### CODE REVIEW

File: tests/__init__.py
Comment (moritzpossecker) [inline tests/__init__.py:1]: ???

File: tests/test_api.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    response = app.test_client().get('/get-teams/fghrt')
+
+    assert response.status_code == 201
+    assert response.data.decode('utf-8') == '[]\n'
+
+def test_get_matches_route():
+    response = app.test_client().get('/get-matches/hhh/hh/5')
+
+    assert response.status_code == 202
+    assert response.data.decode('utf-8') == '[]\n'
```
Comment (moritzpossecker) [inline tests/test_api.py:21]: Perhaps write an additional function to decode data using a certain encoding and then trim it.

File: tests/__init__.py
Comment (MarkusHackspacher) [reply tests/__init__.py:1]: When a directory contains an __init__.py file, Python treats it as a package. This means that modules within the directory can be imported using the dot notation.

File: tests/test_api.py (hunk 1 above)
Comment (MarkusHackspacher) [reply tests/test_api.py:21]: I don´t think need a additional function to decode, is already implemented with the decode('utf-8')  function.

**Comment by MarkusHackspacher** [comment]
```
tests/test_api.py .....                                                                                  [100%]
============================================== 5 passed in 2.20s ===============================================
```

File: tests/test_api.py (hunk 1 above)
Comment (moritzpossecker) [reply tests/test_api.py:32]: No, I meant something like always automatically using the same encoding, if it perhaps at some point has to be changed to UTF-16, and always automatically trimming the response at the same step.

Comment (moritzpossecker) [reply tests/test_api.py:32]: for example

```python
def get_response_data(response) -> string:
    return response.data.decode('utf-8').trim()
```

Comment (MarkusHackspacher) [reply tests/test_api.py:32]: But this example is a good example of not doing that, additional code that needs to be maintained, additional running time. In addition, at some point in the future Chinese characters on a German page will require UTF-16 coding.

File: tests/test_api.py
Comment (moritzpossecker) [inline tests/test_api.py:1]: Nice!!

**Review by moritzpossecker — APPROVED** [review]
Nice work! Thanks a lot

**Pull request merged**